Server/src/models/AuthenticationModel.ts

```diff
@@ -102,7 +102,7 @@ export class AuthenticationModel {
      * This will fetch email, firstname, lastname, organization and dob.
      * @param email - User Email
      */
-    static async getUserDetails(userEmail: string): Promise<IUserDetails> {
+    static async getUserDetails(userEmail: string): Promise<IFetchUserDetail> {
```

Comment:
Can you change this one to fetch aswell instead of get?

Reply:
updated on the latest commit